src/usage/context/usage.tsx

```diff
@@ -109,9 +111,11 @@ export const UsageProvider: FC<Props> = React.memo(({children}) => {
   const [timeRange, setTimeRange] = useState<SelectableDurationTimeRange>(
     DEFAULT_USAGE_TIME_RANGE
   )
-  const {quartzMe} = useSelector(getMe)
 
-  const paygCreditStartDate = quartzMe?.paygCreditStartDate ?? ''
+  const currentIdentity = useSelector(selectCurrentIdentity)
+  const {account} = currentIdentity
+
+  const paygCreditStartDate = account.paygCreditStartDate ?? ''
```

Comment:
`paygCreditStartDate` isn't a required to be returned from the API and therefore isn't required to be present in state. As a result, this guard is still necessary.